specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlServer.json

```diff
@@ -1188,6 +1188,180 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/dedicatedSQLminimalTlsSettings/{dedicatedSQLminimalTlsSettingsName}": {
+      "patch": {
```

Comment:
>patch [](http://example.com/codeflow?start=7&length=5)

you can optionall support PATCH but you must support PUT

Reply:
@pilor  : Pushed the latest changes to use PUT instead of patch operation. Please check